proto/schedule/yaml.go

```diff
@@ -0,0 +1,21 @@
+package schedule
+
+import (
+	"github.com/ghodss/yaml"
+)
+
+func (s *Schedule) MarshalYAML() ([]byte, error) {
```

Comment:
Is there a reason that we need these functions here?  I think that `yaml.Unmarshal` should be able to correctly unmarshal by reading the JSON tags on the field.

Reply:
You'd think so, but because the structs are generated from protos, the `yaml.Unmarshal` function also emits all the extra `XXX_blahblahblah` fields. The workaround uses the proto library's serialize-to-JSON, and then JSON-to-YAML.

Reply:
For the built-in golang `yaml` package I believe that's true, but I thought that the [ghodss/yaml](https://github.com/ghodss/yaml) package was (among other things) a wrapper around `yaml -> json` followed by `json.Marshal` in order to solve this problem without needing clients to convert to JSON first.

We're using `ghodss/yaml` package to serialize proto-generated structs in `kleat` and did not need this workaround.

Reply:
Ahh, you're right - it does the JSON marshalling inside and does not need this file. Removed.